cloudburst/server/scheduler/policy/default_policy.py

```diff
@@ -111,6 +111,7 @@ def pick_executor(self, references, function_name=None):
                 executors.discard(key)
 
         if len(executors) == 0:
+            logging.error('Length of executors is 0.')
```

Comment:
Nit: Probably more explanatory to say that there no available executors.

Reply:
fixed the log.